src/components/DatePicker/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useCallback } from 'react';
+
+import { TextInput } from '../TextInput';
+
+export const DatePicker = ({
+  id,
+  label,
+  value,
+  onChange,
+}: {
+  id: string;
+  label: string;
+  value: string;
+  onChange: (value: string) => void;
+}) => {
+  const onDateChange = useCallback(
+    (value: any) => {
```

Comment:
can you type this as something besides `any`?

Reply:
ahhh, good catch